pykilosort/utils.py

```diff
@@ -62,17 +62,19 @@ def _extend(x, i0, i1, val, axis=0):
 def is_fortran(x):
     if isinstance(x, np.ndarray):
         return x.flags.f_contiguous
-    raise ValueError()
+    raise ValueError('`is_fortran` is only implemented for numpy arrays')
 
 
 def _make_fortran(x):
+    # TODO: unclear - Are we using Dask? Are we explicitly support pykilosort for use with it?
```

Comment:
I started looking at dask but eventually abandoned it -- it's an awesome library but not the right tool for the job here. We can remove the reference here